File: .vscode/Java_Programs/Q3.java
//Problem Statement
//Write a Java program to check whether a number is prime.

import java.util.Scanner;

class PrimeCheck {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n, flag = 0;
        System.out.println("Enter a number: ");
        n = sc.nextInt();

        for(int i = 2; i < n; i++) {
            if(n % i == 0);
                flag = 1; 
        }

        if(flag == 0)
            System.out.println("Prime number");
        else
            System.out.println("Not a prime number");
    }
}

//Expected Output
    //Enter a number: 7
    //Prime number

File: .vscode/C-Programs/Q3.c
//Problem Statement
//Write a program to compute the factorial of a number.


#include <stdio.h>

int main() {
    int n, fact = 1, i;
    printf("Enter a number: ");
    scanf("%d", n); 
    for(i = 1; i < n; i++) 
        fact *= i;
    printf("Factorial is: %d", fact);
    return 0;
}

//Expected Output

    //Enter a number: 5
    //Factorial is: 120
